.eslintrc

```diff
@@ -1,13 +1,29 @@
 {
-  "extends": [
-    "4catalyzer",
-    "prettier",
-    "prettier/flowtype"
-  ],
-  "plugins": [
-    "prettier"
-  ],
+  "plugins": ["prettier"],
+  "extends": ["4catalyzer", "prettier","4catalyzer-typescript"],
+  "env": {
+      "node": true
+  },
   "rules": {
-    "prettier/prettier": "error"
-  }
+    "prettier/prettier": "error",
+    "global-require": "off",
+    "import/no-dynamic-require": "off"
+  },
+  "overrides": [
```

Comment:
lets remove this, and add 4catalyzer-jest to the extends above, i'll update `cli` for this as well